src/index.d.ts

```diff
@@ -568,6 +569,39 @@ declare namespace SDKStandardComponents {
             fatal(arg: unknown): void
         }
     }
+
+    import http from 'http'
+
+    enum RequestResponseType {
+        ArrayBuffer = Symbol('arraybuffer'),
+        JSON = Symbol('json'),
+        Text = Symbol('text'),
+        Stream = Symbol('stream')
+    }
+    enum RequestMethod {
+        GET = 'GET',
+        PATCH = 'PATCH',
+        POST = 'POST',
+        PUT = 'PUT'
+    }
+    interface RequestOptions {
```

Comment:
This is based on the options found here, right? https://github.com/mojaloop/sdk-standard-components/blob/master/src/lib/request.js#L23

Shouldn't qs/querystring be apart of this interface?

Reply:
As well as `responseType`.

Reply:
thanks @kleyow fixed